### PR TITLE
fix loop seek method when input -c option

### PR DESCRIPTION
**Pull request by Pham-Gia-Huong**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -295,7 +295,11 @@ func main() {
 		if config.Query != "" {
 			err = exportRecordsWithQuery(app, config.Fields, writer)
 		} else {
-			err = exportRecordsBySeekMethod(app, writer, config.Fields)
+			fields := config.Fields
+			if len(config.Fields) > 0 {
+				fields = append(fields, "$id")
```
Comment (josh-vo) [inline main.go:317]: I thinks you should check if $id existed in fields.

Comment (Pham-Gia-Huong) [reply main.go:317]: Due $id can't input into config.Fields. We can only check if a second field exists.

Comment (josh-vo) [reply main.go:317]: User can input "... -c $id,...", is it?

Comment (Pham-Gia-Huong) [reply main.go:317]: Yes

**Pull request merged**